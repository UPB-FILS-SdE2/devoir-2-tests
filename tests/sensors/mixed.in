temperature c
temperature k
light
temperature f
exit