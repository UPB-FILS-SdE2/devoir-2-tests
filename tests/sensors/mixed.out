cmd: 24
cmd: 297.15
cmd: 125
cmd: 75.2
cmd: 